AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/Constants.java

```diff
@@ -12,9 +12,12 @@
     public static final int DISPLAYED_REMOTE_PARTICIPANT_SIZE_LIMIT = 3;
 
     /* The key for the meeting name */
-    public static final String GROUP_ID = "GROUP_ID";
+    public static final String JOIN_ID = "JOIN_ID";
 
     /* The key for the Join Call Config */
     public static final String JOIN_CALL_CONFIG = "JOIN_CALL_CONFIG";
 
+    /* The key for the Join Call Type */
+    public static final String CALL_TYPE = "CALL_TYPE";
```

Comment:
We should consider applying these renaming changes in main branch first to minimize changes later on when merging.